### PR TITLE
Bumpversion / CHANGELOG setup

### PR DESCRIPTION
**Pull request by MattF-NSIDC**

### CODE REVIEW

File: dev-requirements.txt [hunk 1]
```diff
@@ -0,0 +1,15 @@
+###############################################################################
+# The runtime dependencies are limited to only the QGIS environment. These dev
+# dependencies only provide support for non-runtime behaviors, e.g.:
+# building docs locally, bumping versions, static analysis, etc.
+###############################################################################
+
+# Documentation:
+sphinx ~=4.3.2
+sphinx-rtd-theme ~=1.0.0
+myst-parser ~=0.15.2
```
Comment (MattF-NSIDC) [inline dev-requirements.txt:10]: These are all included on readthedocs, so we don't need to specify a requirements.txt file in the `help` directory anymore.

Comment (MattF-NSIDC) [reply dev-requirements.txt:10]: Just kidding, apparently `myst_parser` is not.

File: .readthedocs.yaml [hunk 2]
```diff
@@ -17,6 +17,6 @@ sphinx:
 formats:
   - pdf
 
-python:
-  install:
-    - requirements: help/requirements.txt
+    # python:
```
Comment (trey-stafford) [inline .readthedocs.yaml:20]: Do we need to keep this commented bit around?

**Review by trey-stafford — APPROVED** [review]

File: .readthedocs.yaml (hunk 2 above)
Comment (MattF-NSIDC) [reply .readthedocs.yaml:20]: It actually needs to be uncommented. I thought myst_parser was included on RTD, but nah. :(

Comment (MattF-NSIDC) [reply .readthedocs.yaml]: 8bd3e09

File: dev-requirements.txt (hunk 1 above)
Comment (MattF-NSIDC) [reply dev-requirements.txt:10]: 8bd3e09

**Pull request merged**